lib/createAction.lua

```diff
@@ -0,0 +1,45 @@
+--[[
+	A helper function to define a Rodux action creator with an associated name.
+
+	`createAction` provides a utility that makes action creation cleaner
+	and less error prone. Define your Rodux action like this:
+
+		return Action("MyAction", function(value)
+			return {
+				value = value,
+			}
+		end)
+
+	The `type` field will be added automatically. Additionally, the returned action
+	creator now has a 'name' property that can be checked by your reducer:
+
+		local MyAction = require(Reducers.MyAction)

```

Comment:
Isn't the convention to have this be require(Actions.MyAction)?

Reply:
Yeah, I'll have to update these, as well as the wording in the asserts.